proof/infoflow/FinalCaps.thy

```diff
@@ -317,7 +317,7 @@ lemma caps_ref_either_an_object_or_irq':
 
 locale FinalCaps_1 =
   fixes aag :: "'a subject_label PAS"
-  (* FIXME IF: precludes X64 *)
+  (* FIXME IF: precludes x64 *)
```

Comment:
X64 was correct

Reply:
Was basing these off Raf's comments, should I flip those too?

Reply:
not sure what those referred to -- X64 is the L4V_ARCH which this comment was referring to, x64 is the processor architecture in normal text.